src/components/apps/launch/styles.js

```diff
@@ -3,8 +3,31 @@ export default (theme) => ({
 
     // Keeps the content panel at a static height
     // so the nav buttons don't move around below it.
+    stepContainer: {
+        height: "55vh",
+        overflow: "auto",
+    },
     stepContent: {
-        height: "42vh",
+        height: "50vh",
```

Comment:
Ya I haven't optimized this for mobile screens yet. See my notes at the top although am not sure why it would look different in SB.

Reply:
I do see it scrolling in SB but fine when you run the app. Not sure whats the difference :
![image](https://user-images.githubusercontent.com/1250790/88447073-206c6600-ce4d-11ea-947b-34c4050e629c.png)

Reply:
Ahh ya I see it when I set my laptop for 1280x800 resolution. My laptop defaults to 1440 * 900. 😕

Reply:
My browser defaults to 1366x768, and almost the entire nav bar is cutoff at that resolution 😞  in dev mode (not Storybook):
![image](https://user-images.githubusercontent.com/996408/88447215-8cb37f00-cde5-11ea-8792-d59ba80de3a3.png)

Reply:
One more thing to consider when adjusting the size and scroll behavior of these param and review steps:

For #108, I made the `fieldset` scroll with the param groups in the hopes that it would be an indication to users that there was more hidden (especially mac and iOS users that have scroll bars hidden by default), and that they have to scroll to see everything; and when they scroll to the end of the parameters, the bottom outline of the `fieldset` then scrolls into view. I'm not sure if that's intuitive, though.

Reply:
@psarando  - Can you double check your laptop if you have the correct setting in your `Display`. I had to fix the same settings for Nirav as well.

![image](https://user-images.githubusercontent.com/1250790/88548942-5a9c5a00-d03d-11ea-94c1-dd6bb186ebb1.png)

Reply:
If it is set to `Scaled`, then that will cause issues since it is zooming your screen.

Reply:
It is set for `Default for display`, but either way, I don't think we can require this for our users.

Reply:
True. But how did you change your resolution to `1280x800`? Each laptop screen has been built for different resolution range. We cannot build our app for screen that is zoomed by `200%` for example.

Reply:
If the screen is built for `1280 * 800`, thats where We should test if our app is working correctly

Reply:
This is how it looks like at 1280 * 800 when simulating in chrome (at 30% zoom).

![image](https://user-images.githubusercontent.com/1250790/88577132-1ec8bb00-d064-11ea-93b1-87fae1dda04c.png)

Reply:
Clearly, my laptop screen cannot display `1280 * 800`. That will leave elements out of bounds.

Reply:
With my changes in
![image](https://user-images.githubusercontent.com/1250790/88577703-145af100-d065-11ea-8e7c-75ffeb9779f1.png)

Reply:
> True. But how did you change your resolution to `1280x800`?

I didn't adjust my resolution, but I noticed the same effect in Firefox dev mode set at that resolution as I was seeing in the screenshot I gave above (Step 2 scrolls in an app with only 1 parameter).

> Each laptop screen has been built for different resolution range. We cannot build our app for screen that is zoomed by `200%` for example.

I think Apple using the term "Scaled" is causing some confusion (at least for me). I was under the impression that changing those display settings in OSX was changing the actual display resolution, not just zooming the screen. I agree that we can't account for all possible zoomed displays, I was just pointing out that it seemed like something in the CSS or the layout did not look right in some common display resolutions (1366×768 being the most common according to https://www.hobo-web.co.uk/best-screen-size/). I only used that 1280x800 as an example since it was already pointed out before that the current layout of this view didn't look right at that resolution (which is also one of the most common desktop resolutions according to that site above).

Also, regardless of a user's laptop or desktop screen resolution, they may prefer to use a browser in a smaller window, and I don't think we should force them into full-screen mode. I use the browser in a 1366×768 window even though my laptop resolution is set for 1440x900 (my desktop has an even higher resolution but I use the browser in a 1024x768 window there), and I can easily resize the browser to 1280x800 if I wanted.

I thought responsive CSS was supposed to move us beyond "this site looks best in Netscape Navigator in 800x640 resolution" 😆

Reply:
Agreed. I just read this...thought it might be useful - https://superuser.com/questions/153998/whats-the-difference-between-zoom-and-scale-in-gimp#:~:text=Zoom%20does%20not%20change%20image,it%20is%20presented%20on%20screen.&text=Scale%20changes%20physical%20image%20size,it%20will%20take%20128x128%20pixels.
Anyway, I figured out how to set the height without setting it `explicitly` in css. So hopefully, the changes looks good across screen sizes and resolutions. I am still testing for edges cases.